app/views/embed/body/_file.html.erb

```diff
@@ -0,0 +1,83 @@
+<div class='sul-embed-body sul-embed-file'
+      data-sul-embed-theme="<%= viewer.asset_url('file.css') %>"
```

Comment:
Can we drop the `viewer` prepend here? I'm seeing some strange stuff locally when assets change.